tests/functional/behave_features/common/utils/chart_certification.py

```diff
@@ -557,82 +555,84 @@ def process_report(self, update_chart_sha=False, update_url=False, url=None,
-                                report['metadata']['tool']['digests']['chart'] = new_sha_value
-                                logging.info(f"Updated sha value in report: {new_sha_value}")
-
-                            #For updating the report.yaml, for invalid_url sceanrio
-                            if update_url:
-                                logging.info(f"Current chart-uri in report: {report['metadata']['tool']['chart-uri']}")
-                                report['metadata']['tool']['chart-uri'] = url
-                                logging.info(f"Updated chart-uri value in report: {url}")
-
-                            if update_versions:
-                                report['metadata']['tool']['testedOpenShiftVersion'] = tested_version
-                                report['metadata']['tool']['supportedOpenShiftVersions'] = supported_versions
-                                report['metadata']['chart']['kubeversion'] = kube_version
-                                logging.info(f"Updated testedOpenShiftVersion value in report: {tested_version}")
-                                logging.info(f"Updated supportedOpenShiftVersions value in report: {supported_versions}")
-                                logging.info(f"Updated kubeversion value in report: {kube_version}")
-
-                            if update_provider_delivery:
-                                report['metadata']['tool']['providerControlledDelivery'] = provider_delivery
-
-                            if unset_package_digest:
-                                del report['metadata']['tool']['digests']['package']
-
-                        with open(report_path, 'w') as fd:
-                            try:
-                                fd.write(yaml.dump(report))
-                                logging.info("Report updated with new values")
-                            except Exception as e:
-                                raise AssertionError("Failed to update report yaml with new values")
-
-                        #For removing the check for missing check scenario
-                        if missing_check:
-                            logging.info(f"Updating report with {missing_check}")
-                            with open(report_path, 'r+') as fd:
-                                report_content = yaml.safe_load(fd)
-                                results = report_content["results"]
-                                new_results = filter(lambda x: x['check'] != missing_check, results)
-                                report_content["results"] = list(new_results)
-                                fd.seek(0)
-                                yaml.dump(report_content, fd)
-                                fd.truncate()
+                        shutil.copyfile(f'{chart.report_file_path}', f'{report_path}')
+
+                        # try:
```

Comment:
Please remove this commented out code.

Reply:
Removed the commented code in the latest commit 0bd67b9